app/src/common/utils/connectRouter.jsx

```diff
@@ -24,6 +26,13 @@ export const connectRouter = (mapURLParamsToProps = takeAll, queryUpdaters = {},
       const namespacedQuery = namespace
         ? extractNamespacedQuery(pageProperties, namespace)
         : pageProperties;
+      if (!namespacedQuery[SIZE_KEY]) {
```

Comment:
Pagination is not this component's responsibility.
It's better to add corresponding getter functions to `withPagination.jsx`:
`getPageSize => () => { ... }` and so on.

Reply:
Fixed.